src/h5web/visualizations/line/LineVisToolbar.module.css

```diff
@@ -0,0 +1,5 @@
+.toolbar {
+  display: flex;
+  padding: 0 0.5rem;
+  font-size: 0.875rem;
+}
```

Comment:
It might be worthwhile to factor out the generic toolbar styling.

Reply:
You can move the `div.toolbar` up to the parent